server/config.go

```diff
@@ -56,10 +57,8 @@ type Config struct {
 	// Etcd onlys support seoncds TTL, so here is second too.
 	LeaderLease int64 `toml:"lease" json:"lease"`
 
-	// Log level.
-	LogLevel string `toml:"log-level" json:"log-level"`
```

Comment:
For Ansible deployment log-level is specified via command line argument: https://github.com/pingcap/tidb-ansible/blob/master/roles/pd/templates/run_pd.sh.j2

Shall we add a migration support for backward compatibility?